packages/delegation-process/src/services/validators.ts

```diff
@@ -169,3 +171,28 @@ export function assertStampExists<S extends keyof Delegation["stamps"]>(
     throw delegationStampNotFound(stamp);
   }
 }
+
+export const assertEserviceIsDelegable = (eservice: EService): void => {
```

Comment:
I know the flag name is only `isDelegable` (I think we could have named it better, in hindsight...), but let's be more specific on the validator 

```suggestion
export const assertEserviceIsConsumerDelegable = (eservice: EService): void => {
```